datalake/src/main/java/com/sequenceiq/datalake/service/validation/cloudstorage/CloudStorageValidator.java

```diff
@@ -21,28 +26,42 @@
 @Component
 public class CloudStorageValidator {
 
+    private static final String DATALAKE_INTERNAL_ACTOR_CRN = new InternalCrnBuilder(Crn.Service.DATALAKE).getInternalCrnForServiceAsString();
```

Comment:
You may need to use the `IAM` service rather than `DATALAKE` in order to get the internal actor CRN to validate in the UMS. See https://github.com/hortonworks/cloudbreak/pull/7481#discussion_r388555752.